app.js

```diff
@@ -86,16 +87,28 @@ function main (key) {
     window.location = '#' + archive.key.toString('hex')
     updateShareLink()
 
-    var widget = explorer(archive, function (ev, entry) {
-      if (entry.type === 'directory') {
-        cwd = entry.name
-      }
-    })
+    var entries = []
+    var widget
+
+    function update () {
+      var fresh = explorer(archive, entries, function (ev, entry) {
```

Comment:
does the explorer still need `archive` anymore if its just passing in entries? and does this just become a generic folder browser?